File: POO/temas/29/rb/29.1.rb
class Frigo
    def initialize(color:, temperatura:)
        @color = color
        @temperatura = temperatura
    end

    def to_s
        return "Color #{@color} | #{@temperatura}ºC"
    end
end


f1 = Frigo.new(color:"rojo", temperatura:19)
puts f1

f2 = Frigo.new(temperatura:17, color:"blanco")
puts f2

File: POO/temas/29/py/29.1.py
class Frigo:
    def __init__(self, color, temperatura):
        self.color = color
        self.temperatura = temperatura

    def __str__(self):
        return f"Color {self.color} | {self.temperatura}ºC"

f1 = Frigo(color="rojo", temperatura=19)
print( f1 )

f2 = Frigo(temperatura=17, color="blanco")
print( f2 )
